README.md

```diff
@@ -31,32 +31,49 @@ A curated list of awesome things related to [Chart.js](https://www.chartjs.org)
   [pcp](https://github.com/sgratzl/chartjs-chart-pcp) | Adds parallel coordinates plot chart type | ✔ | ✔
   [sankey](https://github.com/kurkle/chartjs-chart-sankey) | Adds sankey diagram chart type | | ✔
   [smith](https://github.com/chartjs/Chart.smith.js) | Adds smith chart type | ✔ |
+  [stacked100](https://github.com/y-takey/chartjs-plugin-stacked100) | Draws 100% stacked bar chart | ✔ | ✔
   [treemap](https://github.com/kurkle/chartjs-chart-treemap) | Adds treemap chart type | ✔ | ✔
   [venn](https://github.com/upsetjs/chartjs-chart-venn) | Adds venn and euler chart type | | ✔
   [word-cloud](https://github.com/sgratzl/chartjs-chart-wordcloud) | Adds word-cloud chart type | | ✔
 
 ## Plugins
 
+### Colors and styles
+
   Name | Description | Chart.js v2 | Chart.js v3
   ---- | ----------- | :--: | :--:
-  [annotation](https://github.com/chartjs/chartjs-plugin-annotation) | Draws lines and boxes on the chart area | ✔ | ✔
   [autocolors](https://github.com/kurkle/chartjs-plugin-autocolors) | Automatic color generation | | ✔
   [colorschemes](https://github.com/nagix/chartjs-plugin-colorschemes) | Enables automatic coloring using predefined color schemes | ✔ | 
+  [gradient](https://github.com/kurkle/chartjs-plugin-gradient) | Easy gradients | | ✔
+  [rough](https://github.com/nagix/chartjs-plugin-rough) | Draws charts in a sketchy, hand-drawn-like style using Rough.js | ✔ |
+  [style](https://github.com/nagix/chartjs-plugin-style) | Provides styling options such as shadow, bevel, glow or overlay effects | ✔ |
+
+### Chart elements
```

Comment:
I'm not a fan of the name "Chart Elements", it's not obvious what it means for new users. Any other suggestions?

Reply:
*Add-ons*? Bad, because it's weird to have "add-ons" under "plugins".
*Enhacements*? Bad, because it's too broad, everything's an enhancement of sorts.
*Complications*? Like, "watch complications". Not sure.

WDYT?

Reply:
"Chart Elements" did catch my eye too, but I did not have any better ideas. Features works for me.

Reply:
_Features_ made me think of _Enhancements_ just spitting it out if it tastes better to others :)

Reply:
*Features* are great!